publish.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euo pipefail
+
+LABEL_PUBLISH="publish-sdk"
+PACKAGE_GLOB=(./packages/lib-*)
+NPM_REGISTRY="https://registry.npmjs.org/"
+NPM_REGISTRY_AUTH_KEY="//registry.npmjs.org/:_authToken"
+LABELS=$(curl -s \
+  -H "Accept: application/vnd.github.v3+json" \
+  "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/pulls/${PULL_NUMBER}" \
+  | jq '.labels | map(.name)')
+HAS_PUBLISH=$(printf '%s\n' "${LABELS}" | jq 'index("'${LABEL_PUBLISH}'")')
+
+if [[ "${HAS_PUBLISH}" != "null" ]]; then
```

Comment:
This will always be non-null isn't it?

Reply:
Yes - specifically jq returns a string `null` when it cannot find the index.